web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -289,192 +291,190 @@ export const DailyViewTable = (props: Props): JSX.Element => {
-                                DECIMAL_PLACES
-                              )}
-                              error={grassCureError}
-                              className={classNameForRow}
-                            ></CalculatedCell>
-                            <CalculatedCell
-                              testid={`${daily?.code}-fire-type`}
-                              value={daily?.fire_type}
-                              error={grassCureError}
-                              className={classNameForRow}
-                            ></CalculatedCell>
-                            <IntensityGroupCell
-                              testid={`${daily?.code}-intensity-group`}
-                              value={daily?.intensity_group}
-                              error={grassCureError}
-                              selected={isRowSelected}
-                            ></IntensityGroupCell>
-                            <TableCell colSpan={2}>
-                              {/* empty cell for spacing (Fire Starts & Prev Level columns) */}
+                      <Table>
+                        <TableBody>
+                          <TableRow>
+                            <TableCell className={classes.noBottomBorder}>
+                              {area.name}
                             </TableCell>
                           </TableRow>
-                        )
-                      })}
-                  </React.Fragment>
-                )
-              })}
+                        </TableBody>
+                      </Table>
+                    </StickyCell>
+                    <TableCell colSpan={19} className={classes.planningArea}></TableCell>
+                    <MeanIntensityGroupRollup
+                      area={area}
+                      dailies={
+                        dailyResult
+                          ? dailyResult.dailies.map(
+                              validatedDaily => validatedDaily.daily
+                            )
+                          : []
+                      }
+                      selectedStationCodes={
+                        result ? result.selected_station_code_ids : []
+                      }
+                      meanIntensityGroup={dailyResult?.mean_intensity_group}
+                    ></MeanIntensityGroupRollup>
+                    <FireStartsCell
+                      areaName={area.name}
+                      fireStarts={dailyResult?.fire_starts}
+                    />
+                    <PrepLevelCell
+                      testid={`daily-prep-level-${area.id}`}
+                      toolTipText=" Incomplete data from WFWX for one or more stations. Please exclude station(s) displaying errors."
+                      prepLevel={dailyResult?.prep_level}
+                    />
+                  </TableRow>
+                  {sortBy(area.stations, station => station.code).map(station => {
```

Comment:
Shouldn't be sorting by station code, should be sorting by station's `order_of_appearance_in_planning_area_list`